### PR TITLE
[MNGSITE-501] Clarify profile jdk activation

### PR DESCRIPTION
**Pull request by kwin**

### CODE REVIEW

File: content/apt/guides/introduction/introduction-to-profiles.apt [hunk 1]
```diff
@@ -127,11 +129,12 @@ mvn groupId:artifactId:goal -P profile-1,profile-2,?profile-3
...
   the build environment. These triggers are specified via an <<<\<activation\>>>>
   section in the profile itself. Currently, this detection is limited to
-  prefix-matching of the JDK version, the presence of a system property or
-  the value of a system property. Here are some examples.
+  matching of the JDK version, matching the operating system,
+  the presence of a system property or the value of a system property. Here are some examples.
 
   The following configuration will trigger the profile when the JDK's
-  version starts with "1.4" (eg. "1.4.0_08", "1.4.2_07", "1.4"):
+  version <starts with> "1.4" (eg. "1.4.0_08", "1.4.2_07", "1.4"), particularly it <won't be active>
```
Comment (gnodet) [inline content/apt/guides/introduction/introduction-to-profiles.apt:136]: `particularly` -> `in particular`

File: content/apt/guides/introduction/introduction-to-profiles.apt [hunk 2]
```diff
@@ -127,11 +129,12 @@ mvn groupId:artifactId:goal -P profile-1,profile-2,?profile-3
   Profiles can be automatically triggered based on the detected state of
   the build environment. These triggers are specified via an <<<\<activation\>>>>
   section in the profile itself. Currently, this detection is limited to
-  prefix-matching of the JDK version, the presence of a system property or
-  the value of a system property. Here are some examples.
```
Comment (gnodet) [inline content/apt/guides/introduction/introduction-to-profiles.apt:131]: `prefix` seems an important word to describe what happens here.  Is that a willingful removal ?

File: content/apt/guides/introduction/introduction-to-profiles.apt [hunk 3]
```diff
@@ -127,11 +129,12 @@ mvn groupId:artifactId:goal -P profile-1,profile-2,?profile-3
...
+  matching of the JDK version, matching the operating system,
+  the presence of a system property or the value of a system property. Here are some examples.
```
Comment (gnodet) [inline content/apt/guides/introduction/introduction-to-profiles.apt:133]: `the presence or value of a system property.`

**Review by gnodet — CHANGES_REQUESTED** [review]

File: content/apt/guides/introduction/introduction-to-profiles.apt (hunk 2 above)
Comment (kwin) [reply content/apt/guides/introduction/introduction-to-profiles.apt:131]: well prefix matching is just one possibility as version ranges are supported as well

File: content/apt/guides/introduction/introduction-to-profiles.apt [hunk 4]
```diff
@@ -127,11 +129,12 @@ mvn groupId:artifactId:goal -P profile-1,profile-2,?profile-3
   Profiles can be automatically triggered based on the detected state of
   the build environment. These triggers are specified via an <<<\<activation\>>>>
   section in the profile itself. Currently, this detection is limited to
-  prefix-matching of the JDK version, the presence of a system property or
-  the value of a system property. Here are some examples.
+  matching of the JDK version, matching the operating system,
```
[the remark this answers is not in the document]
Comment (kwin) [reply content/apt/guides/introduction/introduction-to-profiles.apt]: Fixed in https://github.com/apache/maven-site/pull/346/commits/d93b461cf74b40dc8fb8ca96d8c6f6538be810be.

File: content/apt/guides/introduction/introduction-to-profiles.apt (hunk 3 above)
Comment (kwin) [reply content/apt/guides/introduction/introduction-to-profiles.apt]: Fixed in https://github.com/apache/maven-site/pull/346/commits/d93b461cf74b40dc8fb8ca96d8c6f6538be810be.

File: content/apt/guides/introduction/introduction-to-profiles.apt (hunk 1 above)
Comment (kwin) [reply content/apt/guides/introduction/introduction-to-profiles.apt]: Fixed in https://github.com/apache/maven-site/pull/346/commits/d93b461cf74b40dc8fb8ca96d8c6f6538be810be.

**Comment by jira-importer** [comment]
Resolve #1107

**Pull request merged**